src/utils/components/ListEmptyState/ListEmptyState.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { ReactNode } from 'react';
+import { Trans, useTranslation } from 'react-i18next';
+import { useNavigate, useParams } from 'react-router-dom-v5-compat';
+
+import { K8sResourceCommon, ListPageHeader } from '@openshift-console/dynamic-plugin-sdk';
+import {
+  Button,
+  ButtonVariant,
+  EmptyState,
+  EmptyStateActions,
+  EmptyStateBody,
+  EmptyStateFooter,
+  EmptyStateHeader,
+  EmptyStateIcon,
+} from '@patternfly/react-core';
+import { AddCircleOIcon } from '@patternfly/react-icons';
+import { isEmpty } from '@utils/utils';
+
+import ExternalLink from '../ExternalLink/ExternalLink';
+import Loading from '../Loading/Loading';
+
+type ListEmptyStateProps<T> = {
+  children: ReactNode;
+  data: T[];
+  href: string;
+  kind: string;
+  link: string;
```

Comment:
you have both href and link props, this maybe confusing, can you improve the naming here?

Reply:
sure